client/sites-dashboard-v2/sites-dataviews/index.tsx

```diff
@@ -65,19 +65,21 @@ const DotcomSitesDataViews = ( {
 		// If the user clicks on a row, open the site preview pane by triggering the site button click.
 		const handleRowClick = ( event: Event ) => {
 			const target = event.target as HTMLElement;
-			const row = target.closest( '.dataviews-view-table__row' );
+			const row = target.closest( '.dataviews-view-table__row, .dataviews-view-list__item' );
```

Comment:
The click doesn't work on the `li` element on the list view. I think we can change to `li:has(.dataviews-view-list__item)` to get the row, and then use `row.querySelector('.sites-dataviews__preview-trigger')` to get the preview button.

Reply:
Nice catch! Updated.